src/InputGroup/InputGroup.js

```diff
@@ -276,7 +283,12 @@ InputGroup.propTypes = {
 InputGroup.defaultProps = {
     addonPos: 'after',
     inputType: 'text',
-    inputValue: ''
+    inputValue: '',
+    localizedText: {
+        clearButton: 'Clear',
+        stepDownButton: 'Step down',
+        stepUpButton: 'Step up'
```

Comment:
I wonder if this should simply be `up` and `down`

Reply:
@bcullman are you thinking the actual aria-label that was there before should just be 'up'/'down' or stepUpButton/StepDownButton should be called up/down?